### ISSUE TITLE
Has anybody profiled YoloV3 using GTX1660 and GTX1070?

### ISSUE THREAD

**Issue by asfarley** [AlexeyAB/darknet#3592]
Hi,

I've just built a new PC to run YoloV3 for real-time tracking. The new PC has a GTX1660 6GB and an i5-9400F. 

My older PC has an i5-4460 and a GTX-1070 6GB. 

I'm seeing worse performance on the new PC: 6FPS compared to 12-15FPS on the older PC. 

Possible reason: the GTX1660 has only 1408 CUDA cores, vs 1920 CUDA cores in the GTX1070. So my question is: could this really lead to a 50% drop in FPS? Or should I start looking elsewhere? 

In terms of CPU and GPU usage, both old and new show around 25% CPU usage when my application is running and 5-7% GPU usage.

The new machine is very fresh, just installed Windows and began profiling.

**Comment by asfarley** [comment]
To clarify: I'm not profiling YoloV3 directly, I'm profiling my own application. I do understand that there are areas of my own application that you won't be aware of, but generally-speaking, Yolo frame-processing is the bottleneck in my application. 

I have adjusted settings affecting CPU usage in my application; the FPS seems unresponsive to these settings, which suggests that the bottleneck is not CPU usage, but GPU or something else. 

Additionally: I'm reading the video-file for profiling from an M2 SSD, which should be faster than the magnetic HD on my older PC. So the only spec-reduction on the new PC seems to be a 26% reduction in CUDA cores.

**Comment by AlexeyAB** [comment]
> In terms of CPU and GPU usage, both old and new show around 25% CPU usage when my application is running and 5-7% GPU usage.

* Do you use GPU-Z?
* What command do you use for detection?
* What repository do you use?
* What resolution of your videofile?
* Try to use this repository, compile with GPU=1 CUDNN=1 OPENCV=1, with ./darknet detector demo command and look GPU-usage in the GPU-Z. If everything will good - high GPU usage and high FPS, then the issue in your application.

**Comment by asfarley** [comment]
Ok, not a direct answer to your question, but I swapped the video-cards in my PCs, and the results seem to indicate that the GPU was entirely responsible for the difference in performance. 

When I used my older GTX1070 in the newer PC with the i5-9400F, I got 20FPS. 

It's interesting that the GTX1660 only indicates that it was 6% occupied in Task Manager. Anyway, I'm going to close this now, maybe I will profile with GPU-Z when I get a chance.
